tests/test_fmi3.py

```diff
@@ -81,6 +90,49 @@ def test_load_kind_SE(self, ref_fmu):
         with pytest.raises(FMUException, match = re.escape(msg)):
             load_fmu(ref_fmu, kind = "SE")
 
+    def test_get_model_identifier(self):
+        """Test that model identifier is retrieved as expected."""
+        fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu") # any FMI3 ME would suffice
+        assert fmu.get_identifier() == 'VanDerPol'
+
+    def test_get_get_version(self):
+        """Test that FMI version is retrieved as expected."""
+        fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu") # any FMI3 ME would suffice
+        assert fmu.get_version() == '3.0'
+
+    def test_instantiation(self, tmpdir):
```

Comment:
Thinking of removing this test since we have the test with `initialize` below.

Reply:
I'd say keep it, good to have tests in smaller units.